### ISSUE TITLE
dockerize this

### ISSUE THREAD

**Issue by timothyjlaurent** [paleodictyon/angular-tube#1]
Lets put it on docker!!

**Comment by paleodictyon** [comment]
That's a pretty good idea.  That would allow it to be setup with it's own little VM-like environment right?  So it wouldn't need any system wide dependencies, just once within the docker?

**Comment by pixogen** [comment]
Definitely! I have _scrupulous_ intentions for this code, so being able to use it in docker would make it easier to deploy.
